src/main/java/com/toptrumps/core/Game.java

```diff
@@ -6,13 +6,15 @@
 
     private ArrayList<Player> players;
 
-    public Game() {
+    public Game(int numberOfPlayers) {
+        startNewGame(numberOfPlayers);
         players = new ArrayList<Player>();
     }
 
-    public void startNewGame(int numOfPlayers) {
-        for (int i = 0; i < numOfPlayers; i++) {
-            players.add(new AIPlayer("Player" + (i + 1)));
+    private void startNewGame(int numberOfPlayers) {
```

Comment:
We'll need to know the ```numberOfPlayers``` in many instances... consider making it a class member variable.